<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Molecular Genetics and Cell Biology Group - Publications</title>
  <meta name="description" content="MGCBG -- Publications.">
  <link href="https://fonts.googleapis.com/css?family=Anonymous+Pro|Sanchez|Prompt&display=swap" rel="stylesheet">
  <link rel="stylesheet" href="http://localhost:4000/css/main.css">
  <link rel="canonical" href="http://localhost:4000/publications/">
  <script src="http://localhost:4000/js/jquery.js"></script>
  <script src="http://localhost:4000/js/profile.js"></script>
<!--link rel="shortcut icon" type ="image/x-icon" href="http://localhost:4000/images/favicon.ico"> -->
<link rel="apple-touch-icon" sizes="57x57" href="http://localhost:4000/icons/apple-icon-57x57.png">
<link rel="apple-touch-icon" sizes="60x60" href="http://localhost:4000/icons/apple-icon-60x60.png">
<link rel="apple-touch-icon" sizes="72x72" href="http://localhost:4000/icons/apple-icon-72x72.png">
<link rel="apple-touch-icon" sizes="76x76" href="http://localhost:4000/icons/apple-icon-76x76.png">
<link rel="apple-touch-icon" sizes="114x114" href="http://localhost:4000/icons/apple-icon-114x114.png">
<link rel="apple-touch-icon" sizes="120x120" href="http://localhost:4000/icons/apple-icon-120x120.png">
<link rel="apple-touch-icon" sizes="144x144" href="http://localhost:4000/icons/apple-icon-144x144.png">
<link rel="apple-touch-icon" sizes="152x152" href="http://localhost:4000/icons/apple-icon-152x152.png">
<link rel="apple-touch-icon" sizes="180x180" href="http://localhost:4000/icons/apple-icon-180x180.png">
<link rel="icon" type="image/png" sizes="192x192"  href="/icon/android-icon-192x192.png">
<link rel="icon" type="image/png" sizes="32x32" href="http://localhost:4000/icons/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="96x96" href="http://localhost:4000/icons/favicon-96x96.png">
<link rel="icon" type="image/png" sizes="16x16" href="http://localhost:4000/icons/favicon-16x16.png">
<link rel="manifest" href="http://localhost:4000/icons/manifest.json">
<meta name="msapplication-TileColor" content="#ffffff">
<meta name="msapplication-TileImage" content="http://localhost:4000/icons/ms-icon-144x144.png">
<meta name="theme-color" content="#ffffff">




<style>
	.blocker {
  position: fixed;
  top: 0; right: 0; bottom: 0; left: 0;
  width: 100%; height: 100%;
  overflow: auto;
  z-index: 1;
  padding: 20px;
  box-sizing: border-box;
  background-color: rgb(0,0,0);
  background-color: rgba(0,0,0,0.75);
  text-align: center;
}
.blocker:before{
  content: "";
  display: inline-block;
  height: 100%;
  vertical-align: middle;
  margin-right: -0.05em;
}
.blocker.behind {
  background-color: transparent;
}
.modal {
  display: none;
  vertical-align: middle;
  position: relative;
  z-index: 2;
  max-width: 800px;
  box-sizing: border-box;
  width: 90%;
  background: #fff;
  padding: 15px 30px;
  -webkit-border-radius: 8px;
  -moz-border-radius: 8px;
  -o-border-radius: 8px;
  -ms-border-radius: 8px;
  border-radius: 8px;
  -webkit-box-shadow: 0 0 10px #000;
  -moz-box-shadow: 0 0 10px #000;
  -o-box-shadow: 0 0 10px #000;
  -ms-box-shadow: 0 0 10px #000;
  box-shadow: 0 0 10px #000;
  text-align: left;
}

.modal a.close-modal {
  position: absolute;
  top: -12.5px;
  right: -12.5px;
  display: block;
  width: 30px;
  height: 30px;
  text-indent: -9999px;
  background-size: contain;
  background-repeat: no-repeat;
  background-position: center center;
  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADwAAAA8CAYAAAA6/NlyAAAAAXNSR0IArs4c6QAAA3hJREFUaAXlm8+K00Acx7MiCIJH/yw+gA9g25O49SL4AO3Bp1jw5NvktC+wF88qevK4BU97EmzxUBCEolK/n5gp3W6TTJPfpNPNF37MNsl85/vN/DaTmU6PknC4K+pniqeKJ3k8UnkvDxXJzzy+q/yaxxeVHxW/FNHjgRSeKt4rFoplzaAuHHDBGR2eS9G54reirsmienDCTRt7xwsp+KAoEmt9nLaGitZxrBbPFNaGfPloGw2t4JVamSt8xYW6Dg1oCYo3Yv+rCGViV160oMkcd8SYKnYV1Nb1aEOjCe6L5ZOiLfF120EjWhuBu3YIZt1NQmujnk5F4MgOpURzLfAwOBSTmzp3fpDxuI/pabxpqOoz2r2HLAb0GMbZKlNV5/Hg9XJypguryA7lPF5KMdTZQzHjqxNPhWhzIuAruOl1eNqKEx1tSh5rfbxdw7mOxCq4qS68ZTjKS1YVvilu559vWvFHhh4rZrdyZ69Vmpgdj8fJbDZLJpNJ0uv1cnr/gjrUhQMuI+ANjyuwftQ0bbL6Erp0mM/ny8Fg4M3LtdRxgMtKl3jwmIHVxYXChFy94/Rmpa/pTbNUhstKV+4Rr8lLQ9KlUvJKLyG8yvQ2s9SBy1Jb7jV5a0yapfF6apaZLjLLcWtd4sNrmJUMHyM+1xibTjH82Zh01TNlhsrOhdKTe00uAzZQmN6+KW+sDa/JD2PSVQ873m29yf+1Q9VDzfEYlHi1G5LKBBWZbtEsHbFwb1oYDwr1ZiF/2bnCSg1OBE/pfr9/bWx26UxJL3ONPISOLKUvQza0LZUxSKyjpdTGa/vDEr25rddbMM0Q3O6Lx3rqFvU+x6UrRKQY7tyrZecmD9FODy8uLizTmilwNj0kraNcAJhOp5aGVwsAGD5VmJBrWWbJSgWT9zrzWepQF47RaGSiKfeGx6Szi3gzmX/HHbihwBser4B9UJYpFBNX4R6vTn3VQnez0SymnrHQMsRYGTr1dSk34ljRqS/EMd2pLQ8YBp3a1PLfcqCpo8gtHkZFHKkTX6fs3MY0blKnth66rKCnU0VRGu37ONrQaA4eZDFtWAu2fXj9zjFkxTBOo8F7t926gTp/83Kyzzcy2kZD6xiqxTYnHLRFm3vHiRSwNSjkz3hoIzo8lCKWUlg/YtGs7tObunDAZfpDLbfEI15zsEIY3U/x/gHHc/G1zltnAgAAAABJRU5ErkJggg==');

}

.modal-spinner {
  display: none;
  position: fixed;
  top: 50%;
  left: 50%;
  transform: translateY(-50%) translateX(-50%);
  padding: 12px 16px;
  border-radius: 5px;
  background-color: #111;
  height: 20px;
}

.modal-spinner > div {
  border-radius: 100px;
  background-color: #fff;
  height: 20px;
  width: 2px;
  margin: 0 1px;
  display: inline-block;

  -webkit-animation: sk-stretchdelay 1.2s infinite ease-in-out;
  animation: sk-stretchdelay 1.2s infinite ease-in-out;
}

.modal-spinner .rect2 {
  -webkit-animation-delay: -1.1s;
  animation-delay: -1.1s;
}

.modal-spinner .rect3 {
  -webkit-animation-delay: -1.0s;
  animation-delay: -1.0s;
}

.modal-spinner .rect4 {
  -webkit-animation-delay: -0.9s;
  animation-delay: -0.9s;
}

@-webkit-keyframes sk-stretchdelay {
  0%, 40%, 100% { -webkit-transform: scaleY(0.5) }
  20% { -webkit-transform: scaleY(1.0) }
}

@keyframes sk-stretchdelay {
  0%, 40%, 100% {
    transform: scaleY(0.5);
    -webkit-transform: scaleY(0.5);
  }  20% {
    transform: scaleY(1.0);
    -webkit-transform: scaleY(1.0);
  }
}

</style>

</head>


  <body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation" >
  <div  style="background-color: #191970">
	<div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
	  </button>	
    <a class="navbar-brand" href="http://localhost:4000/">Molecular Genetics & Cell Biology Group</a>
	</div>
	<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li><a href="http://localhost:4000/">Home</a></li>
		<li><a href="http://localhost:4000/team">Members</a></li>
		<li><a href="http://localhost:4000/research">Research</a></li>
		<li><a href="http://localhost:4000/publications">Publications</a></li>
		<li><a href="http://localhost:4000/allnews">News</a></li>
		<li><a href="http://localhost:4000/---">Opportunities</a></li>
		<li><a href="http://localhost:4000/contact">Contact</a></li>
		<li><a href="http://localhost:4000/---">Gallery</a></li>
	  </ul>
	</div>
  </div>
</div>


    <div class="container-fluid">
      <div class="row">
        <div id="gridid" class="col-sm-12">
  <h1 id="publications">Publications</h1>

<div class="row">

  <div class="col-sm-12 clearfix">
    <div class="well" style="background-color: #FFFFE0">
      <pubtit><a href="https://doi.org/10.1038/s41598-020-58300-3">Analysis of Plasmodium falciparum Rh2b deletion polymorphism across different transmission areas [Jan, 2020]</a></pubtit>
      <p><strong> Yaw Aniweh, Jonathan Suurbaar, Collins M Morang’a, Prince B Nyarko, Katherine E Wright, Kwadwo A Kusi, Felix Ansah, Eric Kyei-Baafour, Evelyn Quansah, Jessica Asante, Laty G Thiam, Matthew K Higgins, Gordon A Awandare </strong> 
  <br /><em><font color="grey"> Scientific Reports </font></em></p>
      <p>Recent elucidation of the genetic basis of the Vel blood group system has offered the field of blood transfusion medicine an additional consideration in determining the causes of hemolytic reactions after a patient is transfused. The identification of the SMIM1 gene to be responsible for the Vel blood group allows molecular based tools to be developed to further dissect the function of this antigen. Genetic signatures such as the homozygous 17 bp deletion and the heterozygous 17 bp deletion in combination with other single nucleotide polymorphisms (SNPs) and insertion sequences regulate the expression level of the gene. With this knowledge, it is now possible to study this antigen in-depth.</p>
      <p><!--<img src="http://localhost:4000/images/pubpic/dummy.png" class="img-responsive" width="10%" style="float: left" /> --></p>

      <p><!-- <p><strong><a href="https://doi.org/10.1038/s41598-020-58300-3"></a></strong></p> --></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-12 clearfix">
    <div class="well" style="background-color: #FFFFE0">
      <pubtit><a href="https://doi.org/10.1016/j.parepi.2019.e00101">Investigating a Plasmodium falciparum erythrocyte invasion phenotype switch at the whole transcriptome level [Jan, 2020]</a></pubtit>
      <p><strong> Prince B Nyarko, Sarah J Tarr, Yaw Aniweh, Lindsay B Stewart, David J Conway, Gordon A Awandare </strong> 
  <br /><em><font color="grey"> Scientific Reports </font></em></p>
      <p>Recent elucidation of the genetic basis of the Vel blood group system has offered the field of blood transfusion medicine an additional consideration in determining the causes of hemolytic reactions after a patient is transfused. The identification of the SMIM1 gene to be responsible for the Vel blood group allows molecular based tools to be developed to further dissect the function of this antigen. Genetic signatures such as the homozygous 17 bp deletion and the heterozygous 17 bp deletion in combination with other single nucleotide polymorphisms (SNPs) and insertion sequences regulate the expression level of the gene. With this knowledge, it is now possible to study this antigen in-depth.</p>
      <p><!--<img src="http://localhost:4000/images/pubpic/dummy.png" class="img-responsive" width="10%" style="float: left" /> --></p>

      <p><!-- <p><strong><a href="https://doi.org/10.1016/j.parepi.2019.e00101"></a></strong></p> --></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-12 clearfix">
    <div class="well" style="background-color: #FFFFE0">
      <pubtit><a href="https://doi.org/10.1177/1535370219897393">Cell trace far-red is a suitable erythrocyte dye for multi-color Plasmodium falciparum invasion phenotyping assays [Jan, 2020]</a></pubtit>
      <p><strong> Laty G Thiam, Yaw Aniweh, Evelyn B Quansah, Jacob K Donkor, Theresa M Gwira, Kwadwo A Kusi, Makhtar Niang, Gordon A Awandare </strong> 
  <br /><em><font color="grey"> Experimental Biology and Medicine </font></em></p>
      <p>Plasmodium falciparum erythrocyte invasion phenotyping assays are a very useful tool for assessing parasite diversity and virulence, and for characterizing the formation of ligand–receptor interactions. However, such assays need to be highly sensitive and reproducible, and the selection of labeling dyes for differentiating donor and acceptor erythrocytes is a critical factor. We investigated the suitability of cell trace far-red (CTFR) as a dye for P. falciparum invasion phenotyping assays. Using the dyes carboxyfluorescein diacetate succinimidyl ester (CFDA-SE) and dichloro dimethyl acridin one succinimidyl ester (DDAO-SE) as comparators, we used a dye-dilution approach to assess the limitations and specific staining procedures for the applicability of CTFR in P. falciparum invasion phenotyping assays. Our data show that CTFR effectively labels acceptor erythrocytes and provides a stable fluorescent intensity at relatively low concentrations. CTFR also yielded a higher fluorescence intensity relative to DDAO-SE and with a more stable fluorescence intensity over time. Furthermore, CTFR did not affect merozoites invasion of erythrocytes and was not toxic to the parasite’s intraerythrocytic development. Additionally, CTFR offers flexibility in the choice of combinations with several other DNA dyes, which broaden its usage for P. falciparum erythrocyte invasion assays, considering a wider range of flow cytometers with various laser settings.</p>
      <p><!--<img src="http://localhost:4000/images/pubpic/dummy.png" class="img-responsive" width="10%" style="float: left" /> --></p>

      <p><!-- <p><strong><a href="https://doi.org/10.1177/1535370219897393"></a></strong></p> --></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-12 clearfix">
    <div class="well" style="background-color: #FFFFE0">
      <pubtit><a href="https://doi.org/10.1016/j.parepi.2019.e00101">Recent Advances in the Development of Biosensors for Malaria Diagnosis [Jan, 2020]</a></pubtit>
      <p><strong> Yaw Aniweh, Prince B Nyarko, Evelyn Quansah, Laty Gaye Thiam, Gordon A Awandare </strong> 
  <br /><em><font color="grey"> Parasite epidemiology and control </font></em></p>
      <p>The impact of malaria on global health has continually prompted the need to develop more effective diagnostic strategies that could overcome deficiencies in accurate and early detection. In this review, we examine the various biosensor-based methods for malaria diagnostic biomarkers, namely; Plasmodium falciparum histidine-rich protein 2 (PfHRP-2), parasite lactate dehydrogenase (pLDH), aldolase, glutamate dehydrogenase (GDH), and the biocrystal hemozoin. The models that demonstrate a potential for field application have been discussed, looking at the fabrication and analytical performance characteristics, including (but not exclusively limited to); response time, sensitivity, detection limit, linear range, and storage stability, which are first summarized in a tabular form and then described in detail. The conclusion summarizes the state-of-the-art technologies applied in the field, the current challenges and the emerging prospects for malaria biosensors.</p>
      <p><!--<img src="http://localhost:4000/images/pubpic/dummy.png" class="img-responsive" width="10%" style="float: left" /> --></p>

      <p><!-- <p><strong><a href="https://doi.org/10.1016/j.parepi.2019.e00101"></a></strong></p> --></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-12 clearfix">
    <div class="well" style="background-color: #FFFFE0">
      <pubtit><a href="https://doi.org/10.1016/j.parepi.2019.e00101">SMIM1 at a glance; discovery, genetic basis, recent progress and perspectives [Mar, 2019]</a></pubtit>
      <p><strong> Yaw Aniweh, Prince B Nyarko, Evelyn Quansah, Laty Gaye Thiam, Gordon A Awandare </strong> 
  <br /><em><font color="grey"> Parasite epidemiology and control </font></em></p>
      <p>Recent elucidation of the genetic basis of the Vel blood group system has offered the field of blood transfusion medicine an additional consideration in determining the causes of hemolytic reactions after a patient is transfused. The identification of the SMIM1 gene to be responsible for the Vel blood group allows molecular based tools to be developed to further dissect the function of this antigen. Genetic signatures such as the homozygous 17 bp deletion and the heterozygous 17 bp deletion in combination with other single nucleotide polymorphisms (SNPs) and insertion sequences regulate the expression level of the gene. With this knowledge, it is now possible to study this antigen in-depth.</p>
      <p><!--<img src="http://localhost:4000/images/pubpic/smimi.png" class="img-responsive" width="10%" style="float: left" /> --></p>

      <p><!-- <p><strong><a href="https://doi.org/10.1016/j.parepi.2019.e00101"></a></strong></p> --></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-12 clearfix">
    <div class="well" style="background-color: #FFFFE0">
      <pubtit><a href="https://doi.org/10.1038/s41598-018-24218-0">P. falciparum strains spontaneously switch invasion phenotype in sus. culture [Apr, 2018]</a></pubtit>
      <p><strong> Gordon A. Awandare, Prince B. Nyarko, Yaw Aniweh, Reuben Ayivor-Djanie, José A. Stoute </strong> 
  <br /><em><font color="grey"> Scientific reports </font></em></p>
      <p>P. falciparum strains Dd2 and W2mef have been shown to change from sialic acid (SA)-dependent to SA-independent phenotypes when selected on neuraminidase-treated erythrocytes. Following an observation of increasing ability of Dd2 to invade neuraminidase-treated cells when cultured for weeks, we compared invasion phenotypes of Dd2, W2mef and 3D7 strains of Pf that were cultured with suspended or static. While Static Dd2 and W2mef remained SA-dependent for the entire duration of the investigation, Suspended parasites spontaneously and progressively switched to SA-independent phenotype from week 2 onwards. Our data demonstrates a novel mechanism for inducing the switching of invasion pathways in Pf parasites.</p>
      <p><!--<img src="http://localhost:4000/images/pubpic/switchphen.png" class="img-responsive" width="10%" style="float: left" /> --></p>

      <p><!-- <p><strong><a href="https://doi.org/10.1038/s41598-018-24218-0"></a></strong></p> --></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-12 clearfix">
    <div class="well" style="background-color: #FFFFE0">
      <pubtit><a href="https://doi.org/10.1038/d41586-018-05666-0">Local diagnostics kits for Africa being developed in Ghana [Jul, 2018]</a></pubtit>
      <p><strong> Yaw Aniweh, Prosper Kanyong, Francis Krampa, Gordon A. Awandare </strong> 
  <br /><em><font color="grey"> Nature </font></em></p>
      <p>In our view, building local capacity in diagnostics could help Africa to tackle diseases such as malaria, HIV/AIDS and tuberculosis. We have set up a unit to design and develop diagnostic kits at the University of Ghana’s West African Centre for Cell Biology of Infectious Pathogens (WACCBIP). A robust service to monitor public health and deliver treatment depends on reliable early diagnosis of medical conditions. Africa is generally very limited in its development and deployment of diagnostics systems, however, so these are mostly brought in at high cost from the developed world. Furthermore, the stability and usability of such sensing systems are hampered by poor storage conditions and inadequately trained personnel. Using local platforms such as ours for developing diagnostic sensors and instrumentation will help to meet the continent’s growing demand for them. The hope is that ill health will no longer impede the economic prospects of the continent.</p>
      <p><!--<img src="http://localhost:4000/images/pubpic/dummy.png" class="img-responsive" width="10%" style="float: left" /> --></p>

      <p><!-- <p><strong><a href="https://doi.org/10.1038/d41586-018-05666-0"></a></strong></p> --></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-12 clearfix">
    <div class="well" style="background-color: #FFFFE0">
      <pubtit><a href="https://doi.org/10.1016/j.ijpddr.2017.10.005">Antimalarial activity of Malaria Box compounds against P. falciparum clinical isolates [Dec, 2017]</a></pubtit>
      <p><strong> Jersley D Chirawurah, Felix Ansah, Prince B Nyarko, Samuel Duodu, Yaw Aniweh, Gordon A Awandare </strong> 
  <br /><em><font color="grey"> International Journal for Parasitology; Drugs and Drug Resistance </font></em></p>
      <p>Malaria remains a major cause of childhood deaths in resource-limited settings. In the absence of an effective vaccine, drugs and other interventions have played very significant roles in combating the scourge of malaria. The recent reports of resistance to artemisinin necessitate the need for new antimalarial drugs with novel mechanisms of action. Towards the development of new, affordable and easily accessible antimalarial drugs for endemic regions, the Medicines for Malaria Venture (MMV) assembled a total of 400 active antimalarial compounds called the Malaria Box. The potency and the efficacy of the Malaria Box Compounds have been determined mainly using laboratory strains of P. falciparum.</p>
      <p><!--<img src="http://localhost:4000/images/pubpic/malbox.png" class="img-responsive" width="10%" style="float: left" /> --></p>

      <p><!-- <p><strong><a href="https://doi.org/10.1016/j.ijpddr.2017.10.005"></a></strong></p> --></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<p>   </p>

<!-- ## Full List



  Analysis of Plasmodium falciparum Rh2b deletion polymorphism across different transmission areas [Jan, 2020] <br />
  <em>Yaw Aniweh, Jonathan Suurbaar, Collins M Morang’a, Prince B Nyarko, Katherine E Wright, Kwadwo A Kusi, Felix Ansah, Eric Kyei-Baafour, Evelyn Quansah, Jessica Asante, Laty G Thiam, Matthew K Higgins, Gordon A Awandare </em><br /><a href="https://doi.org/10.1038/s41598-020-58300-3"></a>



  Investigating a Plasmodium falciparum erythrocyte invasion phenotype switch at the whole transcriptome level [Jan, 2020] <br />
  <em>Prince B Nyarko, Sarah J Tarr, Yaw Aniweh, Lindsay B Stewart, David J Conway, Gordon A Awandare </em><br /><a href="https://doi.org/10.1016/j.parepi.2019.e00101"></a>



  Cell trace far-red is a suitable erythrocyte dye for multi-color Plasmodium falciparum invasion phenotyping assays [Jan, 2020] <br />
  <em>Laty G Thiam, Yaw Aniweh, Evelyn B Quansah, Jacob K Donkor, Theresa M Gwira, Kwadwo A Kusi, Makhtar Niang, Gordon A Awandare </em><br /><a href="https://doi.org/10.1177/1535370219897393"></a>



  Recent Advances in the Development of Biosensors for Malaria Diagnosis [Jan, 2020] <br />
  <em>Yaw Aniweh, Prince B Nyarko, Evelyn Quansah, Laty Gaye Thiam, Gordon A Awandare </em><br /><a href="https://doi.org/10.1016/j.parepi.2019.e00101"></a>



  SMIM1 at a glance; discovery, genetic basis, recent progress and perspectives [Mar, 2019] <br />
  <em>Yaw Aniweh, Prince B Nyarko, Evelyn Quansah, Laty Gaye Thiam, Gordon A Awandare </em><br /><a href="https://doi.org/10.1016/j.parepi.2019.e00101"></a>



  P. falciparum strains spontaneously switch invasion phenotype in sus. culture [Apr, 2018] <br />
  <em>Gordon A. Awandare, Prince B. Nyarko, Yaw Aniweh, Reuben Ayivor-Djanie, José A. Stoute </em><br /><a href="https://doi.org/10.1038/s41598-018-24218-0"></a>



  Local diagnostics kits for Africa being developed in Ghana [Jul, 2018] <br />
  <em>Yaw Aniweh, Prosper Kanyong, Francis Krampa, Gordon A. Awandare </em><br /><a href="https://doi.org/10.1038/d41586-018-05666-0"></a>



  Antimalarial activity of Malaria Box compounds against P. falciparum clinical isolates [Dec, 2017] <br />
  <em>Jersley D Chirawurah, Felix Ansah, Prince B Nyarko, Samuel Duodu, Yaw Aniweh, Gordon A Awandare </em><br /><a href="https://doi.org/10.1016/j.ijpddr.2017.10.005"></a>


-->

</div>

      </div>
    </div>

    <div id="footer" class="panel" style="background-color: #191970;">
  <div class="panel-footer" style="background-color: #DAA520;">
	<div class="container-fluid">
	  <div class="row">
		<div class="col-sm-4">
			
		  <p>&copy 2019 Awandare Lab. Site made with <a href="https://jekyllrb.com">Jekyll</a>;</p>
              <p>We are part of the <a href="http://waccbip.org">West African Centre for Infectious Diseases</a> at <a href="https://www.ug.edu.gh/">University of Ghana</a>.</p>
            

		   <p>  </p><p>
            		        
		</div>
	<!--
		<div class="col-sm-4">
		  Funding:<br />
		  - <a href="http://www.nwo.nl/en/research-and-results/programmes/Talent+Scheme">Vidi </a> and <a href='https://www.fom.nl/en/news/press-releases/2016/11/18/28634/'>Projectruimte</a> grants from <a href="http://www.nwo.nl">NWO</a> <br />
		  - <a href="https://www.universiteitleiden.nl/en/research/research-projects/science/frontiers-of-nanoscience-nanofront">Frontier of Nanosciences</a>, a gravity program from <a href="http://www.nwo.nl">NWO</a>
          - <a href='https://www.universiteitleiden.nl/en/news/2017/08/two-erc-grants-for-leiden-physics'>ERC starting grant</a>
		</div>
	-->
		<div class="col-sm-4">
		  Contact:<br />
		  Department of Biochemistry, Cell & Molecular Biology<br />
		  Volta Road<br />
		  P.O. Box LG 54<br />
		  Legon-Ghana<br />
           <!-- (<a href="https://goo.gl/maps/H2RjutkyoUH2">Maps</a>, <a href="http://www.physics.leidenuniv.nl/travel-directions">Directions</a>)-->
		</div>
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="http://localhost:4000/js/bootstrap.min.js"></script>


  </body>

</html>
